etc/scripts/create-new-schema.sh

```diff
@@ -21,5 +21,4 @@ pg_dump -U uta_admin -h localhost -d uta -n "$source_uta_v" | \
 # create new schema
 gzip -cdq $dumps_dir/"$source_uta_v".pgd.gz | \
  sbin/pg-dump-schema-rename "$source_uta_v" "$dest_uta_v" | \
- sbin/pg-dump-schema-rename "uta_1_1" "$dest_uta_v" | \
```

Comment:
This one time update to remove "uta_1_1" from the psql dump can be moved to `misc/gene-update/upgrade-uta-schema.sh`.